src/server/worker/master.go

```diff
@@ -1093,3 +1115,16 @@ func now() *types.Timestamp {
 	}
 	return t
 }
+
+var commitNotFoundRe = regexp.MustCompile("commit [^ ]+ not found in repo [^ ]+")
+var commitDeletedRe = regexp.MustCompile("commit [^ ]+ deleted")
```

Comment:
Similar to the comment on the other PR, it might be nice to have `IsCommitDeleted` and `IsCommitNotFound` functions in `src/server/pfs/pfs.go` so that we have a canonical way to recognize these errors.

Reply:
Done